packages/components/src/components/scrollable/scrollable.stories.ts

```diff
@@ -43,148 +47,147 @@ export default {
 };
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
-export const ButtonWithGradient = {
-  parameters: { controls: { exclude: ['buttons'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      args,
-      constants: [{ type: 'attribute', name: 'buttons', value: true }]
-    });
-  }
-};
-
-export const Shadow = {
-  parameters: { controls: { exclude: ['shadows'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      args,
-      constants: [{ type: 'attribute', name: 'shadows', value: 'true' }]
-    });
-  }
-};
-
-export const Scrollbar = {
-  parameters: { controls: { exclude: ['scrollbars'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      args,
-      constants: [{ type: 'attribute', name: 'scrollbars', value: 'true' }]
-    });
-  }
-};
+/**
+ * Use the `orientation` attribute to set the scroll direction of the component.
+ */
 
-export const CustomIcon = {
-  parameters: {
-    controls: {
-      exclude: ['default', 'buttons', 'icon-start', 'icon-end']
-    }
-  },
-  render: () => {
-    return html`
-      <sd-scrollable buttons>
-        <div
-          class="slot slot--border slot--text items-start"
-          style="height:max-content; width:max-content; padding: 1rem; justify-content:start;"
-        >
-          <p>Scroll and give it a try!</p>
+export const Orientation = {
+  name: 'Orientation',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-scrollable orientation="horizontal">
+        <div class="text-sm items-start h-max w-max p-4">
+          <p>Scroll horizontally</p>
           <br />
           <p>This is a long scrollable content.</p>
```

Comment:
Use same text like in Figma

Union Investment has extended its lease with Converse, the footwear and apparel brand known for its deep connection to youth culture, at 1 Lovejoy Wharf in Boston. The contract for around 20,000 sqm of state-of-the-art office space was renewed long-term. The building has been part of the UniImmo: Global open-ended real estate fund portfolio since 2016 and is the global headquarters of the lifestyle brand, which has been part of NIKE, Inc. since 2003.